### PR TITLE
Explain local photonlib build usage

### PR DESCRIPTION
**Pull request by amquake**

### CODE REVIEW

File: source/docs/other/contributing/photonvision/build-instructions.rst [hunk 1]
```diff
@@ -92,3 +94,51 @@ Running the following command under the root directory will build the jar under
...
+      ``./gradlew generateVendorJson``
+
+   .. tab-item:: Windows (cmd)
+
+      ``gradlew generateVendorJson``
+
+This generates a vendordep JSON of your local build
+at :code:`photon-lib/build/generated/vendordeps/photonlib.json`.
+
+The photonlib source can be published to your local maven repository
```
[the remark this answers is not in the document]
Comment (amquake) [reply source/docs/other/contributing/photonvision/build-instructions.rst]: just didn't fit in my IDE due to zoom

**Comment by amquake** [comment]
Not really familiar with rtd/sphinx, why is `tab-set` used? I see that you can use grouped tabs with `tabs` and code blocks with `code-tab`, but I couldn't easily figure out how to apply the same styling that `tab-set` currently uses.

**Comment by mdurrani808** [comment]
https://github.com/executablebooks/sphinx-design/blob/main/docs/snippets/rst/tab-basic.txt
Tab-set is the syntax for sphinx-design tabs. We used to use different tabs so there is some inconsistencies.
https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html. you should be able to just specify a language attribute in a code block.

**Comment by amquake** [comment]
> you should be able to just specify a language attribute in a code block.

Are you referring to the inline `:code:` blocks? I realized what I wanted was just double backticks ` `` `` `.

**Review by mdurrani808 — APPROVED** [review]
Thanks!

**Pull request merged**